src/MauiIcons.Core/Extensions/BaseIconExtension.cs

```diff
@@ -96,7 +70,7 @@ Binding SetFontProperties(object targetObject, bool disableConverter = false)
         {
             case Button button:
                 button.FontFamily = Icon.GetFontFamily();
-                button.SetBinding(Button.TextColorProperty, new Binding(nameof(IconColor), source: this, mode: BindingMode.OneWay, 
+                button.SetBinding(Button.TextColorProperty, new Binding(nameof(IconColor), source: this, mode: BindingMode.OneWay,
```

Comment:
We can use `OneTime` here instead of `OneWay`

Reply:
> We can use `OneTime` here instead of `OneWay`

I'll look into it 👍